zbx_clickhouse_monitor.sh

```diff
@@ -57,7 +60,7 @@ function run_ch_query()
 	DATABASE="system"
 
 	SQL="SELECT value FROM ${DATABASE}.${TABLE} WHERE $COLUMN = '$METRIC'"
-	/usr/bin/clickhouse-client -h "$CH_HOST" -d "$DATABASE" -q "$SQL"
+	/usr/bin/clickhouse-client -h "$CH_HOST" -d "$DATABASE" -q "$SQL" $(echo "$ADD_FLAGS")
```

Comment:
Why do we need to `$(echo "$ADD_FLAGS")`, why not simply `"$ADD_FLAGS"`?

Reply:
That was my first try. Looks like clickhouse-client assumes that string in quotes is one argument only. So, there is no problem with `/usr/bin/clickhouse-client -h localhost -d default "-c"` but produces an error for `/usr/bin/clickhouse-client -h localhost -d default "-c -u user --password something"`

Reply:
Maybe "echo" is not the best way of doing this, but at least one of them.

Reply:
ok, it's easier to just remove quotes, without it works fine, and looks safe for injection. Let's discuss the issue above and i'll update PR